scripts/offload-idea.js

```
@@ -0,0 +1,84 @@
+      inQuotes = false;
+      textParts.push(arg.slice(0, -1));
+      text = textParts.join(' ');
+    } else if (inQuotes) {
+      textParts.push(arg);
+    } else if (text === '') {
+      text = arg;
+    }
+  }
+}
+
+const argv = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'argv'. Consider to delete it or rename it.

'argv' is defined here:


(lint/suspicious/noRedeclare)